src/components/student/CourseEnroll.js

```diff
@@ -1,19 +1,124 @@
 import React, {useState, useEffect} from 'react';
+import Button from '@mui/material/Button';
+import {confirmAlert} from "react-confirm-alert";
 
 // students displays a list of open sections for a 
 // use the URL /sections/open
 // the REST api returns a list of SectionDTO objects
 
 // the student can select a section and enroll
-// issue a POST with the URL /enrollments?secNo= &studentId=3
+// issue a POST with the URL /enrollments?secNo= &studentId=3  -RH this does not work. Tests in postman says method post not supported
+//  is this supposed to be  @PostMapping("/enrollments/sections/{sectionNo}")
+// using for now, checking with prof
```

Comment:
I think this is correct. You need `@PostMapping("/enrollments/sections/{sectionNo}")` with a `studentId` query parameter. example: `"/enrollments/sections/7?studentId=3"

Reply:
I agree.  Just waiting for response from Prof to confirm.